build/circle-deps.sh

```diff
@@ -85,3 +107,48 @@ grep -v '^cmd' GLOCKFILE | glock sync -n
 # Recursively invoke this script inside the builder docker container,
 # passing "docker" as the first argument.
 $(dirname $0)/builder.sh $0 docker
+
+# According to
+# https://discuss.circleci.com/t/cache-save-restore-algorithm/759, the
+# cache is only collected from container (shard) 0. We workaround this
```

Comment:
i think "work around" is two words, when used as a verb (here and in commit message)